druntime/test/exceptions/src/cpp_demangle.d

```diff
@@ -0,0 +1,19 @@
+import core.demangle;
+
+extern(C) char* __cxa_demangle (const char* mangled_name,
+                                                char* output_buffer,
+                                                size_t* length,
+                                                int* status) nothrow pure @trusted;
+
+extern (C++) void thrower(int a) {
+    throw new Exception("C++ ex");
+}
+void caller() {
+    thrower(42);
+}
+
+void main()
+{
+    caller();
+    __cxa_demangle(null, null, null, null);
+}
```

Comment:
This test should be testing the output, but I don't see anything of the sort?

Reply:
makefile does that.

https://github.com/dlang/dmd/pull/14598/files#diff-d6356b70d1cf336e36223b63cee4009b1b40e70e963cf316980c163e01982ff2R81